examples/typescript/modules/typescript.js

```diff
@@ -1,11 +1,30 @@
 module.exports = function (options, next) {
   // Extend build
   this.extendBuild((config) => {
+    const babelOptions = JSON.stringify(R.merge(this.options.build.babel, {
```

Comment:
good catch, it's bad copy past from my project, i will do the change to not use ramda.